lib/Model/Order.php

```diff
@@ -480,9 +487,39 @@ public function listInvalidProperties()
         if ($this->container['id'] === null) {
             $invalidProperties[] = "'id' can't be null";
         }
+        if ($this->container['order_state'] === null) {
+            $invalidProperties[] = "'order_state' can't be null";
+        }
+        if ($this->container['market_id'] === null) {
+            $invalidProperties[] = "'market_id' can't be null";
+        }
+        if ($this->container['eshop_id'] === null) {
+            $invalidProperties[] = "'eshop_id' can't be null";
+        }
+        if ($this->container['additional_costs'] === null) {
+            $invalidProperties[] = "'additional_costs' can't be null";
+        }
+        if ($this->container['parcels'] === null) {
+            $invalidProperties[] = "'parcels' can't be null";
+        }
+        if ($this->container['products'] === null) {
+            $invalidProperties[] = "'products' can't be null";
+        }
+        if ($this->container['recommended_products'] === null) {
+            $invalidProperties[] = "'recommended_products' can't be null";
+        }
+        if ($this->container['files'] === null) {
+            $invalidProperties[] = "'files' can't be null";
+        }
         if ($this->container['links'] === null) {
             $invalidProperties[] = "'links' can't be null";
         }
+        if ($this->container['external_links'] === null) {
+            $invalidProperties[] = "'external_links' can't be null";
+        }
+        if ($this->container['communication'] === null) {
```

Comment:
Chápu tohle správně tak že se to "zvaliduje" když existuje objekt communication ale je prázdný, ale když ho vůbec nenastavíš tak se nic nestane (já jen že v api je nepovinný).

Reply:
co jsem na to koukal, tak je to validace prichoziho objektu Order a nejsem si jisty ze se tenhle check pouziva pokud ho sam nepouzijes

Reply:
super, díky za info 👍